npm/webpack-dev-server/src/loader.ts

```diff
@@ -49,7 +50,11 @@ function buildSpecs (projectRoot: string, files: Cypress.Cypress['spec'][] = [])
 }
 
 // Runs the tests inside the iframe
-export default function loader (this: CypressCTWebpackContext) {
+export default function loader (this: CypressCTWebpackContext & LoaderContext<any>) {
+  // In Webpack 5, a spec added after the dev-server is created won't
+  // be included in the compilation. Disabling the caching of this loader ensures
+  // we regenerate our specs and include any new ones in the compilation.
+  this.cacheable(false)
```

Comment:
Do we want this to be conditional (if we are using Webpack 5)? Does `this.cacheable` even exist in Webpack 4?

Reply:
It won't throw even if it doesn't. I'm fine with this.

Reply:
`this.cacheable` is valid for Webpack 4 and 5
https://v4.webpack.js.org/api/loaders/#thiscacheable